contents/blog/the-posthog-array.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2023-02-24
+title: "The PostHog Changelog"
```

Comment:
```suggestion
title: "PostHog Changelog: February 24, 2023"
```

It's preferable to have the date in the title. Updating the title is a useful signal to get Google to re-crawl the page when a page is updated. Given we'll be doing this weekly, it's worth doing.

Reply:
OK, sounds good